guides/common/modules/proc_resolving-package-dependencies.adoc

```diff
@@ -16,21 +16,3 @@ This might require some level of testing to determine what dependencies are requ
 If you add a filter that excludes some packages that are required and the Content View has dependency resolution enabled, {Project} ignores the rules you create in your filter in favor of resolving the package dependency.
 
 If you create a content filter for security purposes, to resolve a package dependency, {Project} can add packages that you might consider insecure.
-
-.Procedure
-
-To resolve package dependencies by default, complete the following steps:
-
-. In the {ProjectWebUI}, navigate to *Administer* > *Settings* and click the *Content* tab.
-. Locate the *Content View Dependency Solving Default*, and select *Yes*.
-
-You can also set the default level of dependency resolution that you want.
-You can select between adding packages to solve dependencies only if the required package does not exist, or to add the latest packages to resolve the dependency even if the package exists in the repository.
-
-To set the default level of dependency resolution, complete the following steps:
-
-. In the {ProjectWebUI}, navigate to *Administer* > *Settings* and click the *Content* tab.
```

Comment:
I think there was an error maybe? You deleted the procedure and left the part which you also copied into the new file. Also, the two files have the same anchor so you will have to resolve that.

Reply:
> I think there was an error maybe? You deleted the procedure and left the part which you also copied into the new file. Also, the two files have the same anchor so you will have to resolve that.

I'll just delete the proc file if the changes are green-lighted, so no hustle with the IDs

Reply:
I see. My apologies.

Reply:
I am not following. Has this content completely been moved to `guides/common/modules/con_resolving-package-dependencies.adoc`?